### DISCUSSION TITLE
Grafana 8.3.2

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#6558] [General] [answered]
I see that in the .100 release of SO2, that Grafana will be upgraded to 8.3.1. However, we received notice today of a vulnerability in all version of grafana prior to 8.3.2.

Will the .100 release be upgraded to 8.3.2 instead of 8.3.1?

Thanks

**Comment by dougburks** [comment, marked as answer]
I've re-opened https://github.com/Security-Onion-Solutions/securityonion/issues/6321 and changed it to Grafana 8.3.2.

**Reply by dougburks** [reply]
Merged into the dev branch for .100:
https://github.com/Security-Onion-Solutions/securityonion-image/pull/163
